data/migrations/versions/10d34e6536f8_create_proxy_cache_config_table.py

```diff
@@ -0,0 +1,38 @@
+"""create proxy_cache_config table
+
+Revision ID: 10d34e6536f8
+Revises: 909d725887d3
+Create Date: 2022-01-13 16:10:10.155558
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '10d34e6536f8'
+down_revision = '909d725887d3'
+
+import sqlalchemy as sa
+
+
+def upgrade(op, tables, tester):
+    op.create_table(
+        "proxycacheconfig",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("user_id", sa.Integer(), nullable=False),
+        sa.Column("creation_date", sa.DateTime(), nullable=False),
+        sa.Column("upstream_registry", sa.String(length=2048), nullable=False),
+        sa.Column("upstream_registry_namespace", sa.String(length=255), nullable=True),
+        sa.Column("upstream_registry_username", sa.String(length=2048), nullable=True),
+        sa.Column("upstream_registry_password", sa.String(length=2048), nullable=True),
+        sa.Column("staleness_period_s", sa.Integer(), server_default="0"),
+        sa.Column("quota_enabled", sa.Boolean(), server_default="0"),
+        sa.ForeignKeyConstraint(
+            ["user_id"],
+            ["user.id"],
+            name=op.f("fk_proxy_cache_config_user_id"),
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_proxy_cache_config"))
+    )
+
+
+def downgrade(op, tables, tester):
+    pass
```

Comment:
Should this remove the table entirely? Not sure if we tend to skip the `downgrade` or not, but in this case it should be easy enough 🤔

Reply:
[downgrade](https://sqlalchemy-migrate.readthedocs.io/en/v0.7.2/versioning.html#:~:text=creating%20a%20table).-,downgrade,-()%20should%20reverse%20changes) should reverse changes made by upgrade. I've added `drop_table` here for now.